content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Hardware recommendations and considerations {#windows-hardware-recommendations}
+
+{{< note >}}
+The following hardware requirements are _recommended_ values and do not represent
+minimum requirements. Refer to [Production worker nodes Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/#production-worker-nodes)
+for guidance on deciding on resources for production worker nodes.
+{{< /note >}}
+
+- 64-bit processor 2 CPU cores or more, capable of supporting virtualization
+- 8GB or more of RAM
+- 50GB or more of free disk space
+
+Refer to the specific Windows OS documentation for the most
+up-to-date information on minimum hardware requirements. For Windows Server
+installations, refer to
+[Hardware requirements for Windows Server Microsoft documentation](https://learn.microsoft.com/en-us/windows-server/get-started/hardware-requirements)
+for the bare minimum hardware requirements. 
+
+To optimize system resources, if a graphical user interface is not required,
+it may be preferable to use a Windows OS installation that excludes
+the [Windows Desktop Experience](https://learn.microsoft.com/en-us/windows-server/get-started/install-options-server-core-desktop-experience)
+installation option, as this configuration typically frees up more system 
+resources. 
+
+When it comes to considering disk space, compared to Linux worker nodes, Windows images typically exceed the sizes of
+Linux images, with sizes ranging
+from [300MB to as much 11GB]((https://techcommunity.microsoft.com/t5/containers/nano-server-x-server-core-x-server-which-base-image-is-the-right/ba-p/2835785))
+for a single image. Additionally, take note that the `C:` drive in a Windows container represents a virtual free size of
+20GB on the host. See 
```

Comment:
```suggestion
20GB (by default) on the host. See 
```

It is possible to specify the size of the `C:` drive in Windows containeres but this is not yet wired-up in K8s (see https://github.com/kubernetes/enhancements/issues/3746)

Reply:
Interesting, thanks for sharing! Including that.

Reply:
Have incorporated all the latest feedback, kindly take another look @marosset